src/main/java/net/minecraftforge/client/model/generators/ExistingFileHelper.java

```diff
@@ -0,0 +1,54 @@
+package net.minecraftforge.client.model.generators;
+
+import com.google.common.base.Preconditions;
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+import net.minecraft.data.DataGenerator;
+import net.minecraft.resources.ResourcePackType;
+import net.minecraft.util.ResourceLocation;
+
+import java.io.IOException;
+import java.nio.file.*;
+import java.util.Collection;
+import java.util.List;
+import java.util.function.Function;
+
+public class ExistingFileHelper {
+    public static ExistingFileHelper INSTANCE;
```

Comment:
Do not like the use of global state here, this should be passed around instead of reassigned.